### PR TITLE
Audit3 removed content posts and reactions

### LINKED ISSUE #3222: Removing runtime native channel comments, reactions and channel discussion boards
Has to be after this has been positively resolved https://github.com/Joystream/joystream/issues/2932.

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201958687417145/1201958693727329) by [Unito](https://www.unito.io)

### PR DESCRIPTION
**Pull request by dobertRowneySr**
fixes #3222

### CODE REVIEW

**Review by bedeho — APPROVED** [review]
LGTM.

File: runtime-modules/content/src/tests/mock.rs [hunk 1]
```diff
@@ -536,7 +518,7 @@ pub fn get_open_auction_params() -> OpenAuctionParams<Test> {
 // membership trait implementation and related stuff
 
 parameter_types! {
-    pub const ExistentialDeposit: u32 = 0;
+    pub const ExistentialDeposit: u32 = 1;
```
Comment (Lezek123) [inline runtime-modules/content/src/tests/mock.rs]: Why was this value changed?

**Review by Lezek123 — CHANGES_REQUESTED** [review]
Two places where I noticed some remainings of the removed feature:
- `VideoPostDoesNotExist` in `content/src/errors.rs`
- `BloatBondCap` in `content/src/tests/mock.rs`

Also, all the stuff related to video comments' `moderators` should be removed as agreed.

I noticed that tests are failing on this branch:
```
error[E0599]: no method named `with_transfer_status_by_member_id` found for struct `fixtures::UpdateChannelTransferStatusFixture` in the current scope
    --> runtime-modules/content/src/tests/moderators.rs:103:14
     |
103  |             .with_transfer_status_by_member_id(DEFAULT_MEMBER_ID)
     |              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ method not found in `fixtures::UpdateChannelTransferStatusFixture`
     | 
    ::: runtime-modules/content/src/tests/fixtures.rs:1842:1
     |
1842 | pub struct UpdateChannelTransferStatusFixture {
     | --------------------------------------------- method `with_transfer_status_by_member_id` not found for this
```
Could you look into that?

File: runtime-modules/content/src/tests/mock.rs (hunk 1 above)
Comment (dobertRowneySr) [reply runtime-modules/content/src/tests/mock.rs]: To accomodate both  Storage and membership traits.
I updated it with two different contstants now, so `ExistentialDeposit` is back to `0`

**Comment by Lezek123** [comment]
A few more remainings uncovered:

**`runtime/lib`:**

```rust
    pub const CleanupMargin: u32 = 3;    // TODO: update
    pub const CleanupCost: u32 = 1; // TODO: update
```
    
**`runtime-modules/content/src/tests/mock.rs`**:
 ```rust
    pub const DEFAULT_MODERATOR_ACCOUNT_ID: u64 = 105;
```

```rust
    pub const DEFAULT_MODERATOR_ID: u64 = 205;
```

```rust
    pub const MaxModerators: u64 = 5;
    pub const CleanupMargin: u32 = 3;
    pub const CleanupCost: u32 = 1;
```

```rust
    pub const VideoCommentsModuleId: ModuleId = ModuleId(*b"m0:forum"); // module : forum
```

**Review by Lezek123 — APPROVED** [review]
LGTM!

**Pull request merged**